pyproject.toml

```diff
@@ -54,7 +54,7 @@ graphviz = "^0.16"
 jupyterlab = "^3.0.14"
 keyring = "^23.0.1"
 py-spy = "^0.3.4"
-pystac = "^0"
+pystac = "^1"
```

Comment:
Should we be any stricter on minor versions? Or is any 1.x fine? Probably is fine, I'm just checking.

Reply:
less restrictive seems best, especially with pystac as a 'dev' dependency (although might be worth getting more specific if it becomes a core dependency #20)? If I understand correctly these dev pins are only considered when running `poetry update` where the lockfile is solved for?

Reply:
Yes I think that's right. `poetry update` gives you the latest versions of everything within the bounds specified in the lockfile. Agreed that we should leave it like this for now, and we'll get more restrictive if necessary in the future.